setup.cfg

```diff
@@ -0,0 +1,23 @@
+[flake8]
+exclude = migrations
```

Comment:
Added some sensible defaults here, feel free to discuss what these values should be, like line length for example etc.